url_handlers.go

```diff
@@ -96,6 +96,33 @@ func WaitListHandler(w http.ResponseWriter, r *http.Request) {
   RenderTemplate(w, "assets/templates/waitlist.html.tmpl", nil)
 }
 
+func isAssignedBuzzerHandler(w http.ResponseWriter, r *http.Request) {
+    returnObj := map[string] interface{} {"status": false}
+
+    if !IsUserLoggedIn(GetSession(w, r)) {
+        w.WriteHeader(401)
+        returnObj["error_message"] = "Request unauthorized"
+    } elif r.Method == "POST" {
+        active_party_id = r.FormValue("active_party_id")
+        var activeparty ActiveParty
+        db.First(&activeparty, active_party_id)
+
+        if (activeparty.BuzzerID != nil) {
```

Comment:
Should probably checker whether or not `db.First()` actually found an active party with the provided ID (by checking whether or not `activeparty` is empty) otherwise send back an error_message saying that a party with that id doesn't exist.